index.html

```diff
@@ -685,6 +719,26 @@ <h3>Converting between <code>tiltX</code> / <code>tiltY</code> and <code>altitud
 </pre>
             </section>
         </section>
+        <section>
+            <h2><code>DeviceProperties</code> interface</h2>
+            <pre class="idl">
+                dictionary DevicePropertiesInit {
+                    long uniqueId = 0;
```

Comment:
The default should be -1 I think, right? I.e. by default we would reflect that the generating device couldn't be identified.